src/views/clusteroverview/SettingsTab/settings-tab.scss

```diff
@@ -14,6 +14,11 @@
     padding: var(--pf-t--global--spacer--2xl);
     padding-left: calc(var(--pf-t--global--spacer--3xl) * 2);
   }
+
+
+  &--indented {
+    margin-left: 0.8rem;
```

Comment:
did you find a patternfly variable that its good for this ?  Like `--pf-t--global--spacer--sm` ? 
https://www.patternfly.org/design-foundations/spacers/#patternfly-spacers